apps/air-discount-scheme/backend/src/app/modules/user/user.service.ts

```diff
@@ -29,52 +18,15 @@ export class UserService {
   constructor(
     private readonly flightService: FlightService,
     private readonly nationalRegistryService: NationalRegistryService,
-    private readonly nationalRegistryIndividualsApi: EinstaklingarApi,
```

Comment:
The EinstaklingarAPI came from the NationalRegistryClientModule if I'm not mistaken. If that isn't used anymore then I think the following changes should happen to `user.module.ts`

```diff
...
-import { NationalRegistryClientModule } from '@island.is/clients/national-registry-v2'
import { CacheModule } from '../cache'
import { PrivateUserController } from './user.controller'

@Module({
  imports: [
    FlightModule,
    NationalRegistryModule,
-    NationalRegistryClientModule,
    CacheModule,
  ],
...
```

Reply:
Great catch!